File: answers/IT0506/conditionals/Date.java
import java.util.Scanner;

public class Date {

	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		int d=sc.nextInt();
		int m=sc.nextInt();
		int y=sc.nextInt();
		switch(m) {
		case 01:
			System.out.print("January"+" "+d+" "+y);
			 break;
		case 02:
			System.out.print("February"+" "+d+" "+y);break;
		case 03:
			System.out.print("March"+" "+d+" "+y);break;
		case 04:
			System.out.print("April"+" "+d+" "+y);break;
		case 05:
			System.out.print("May"+" "+d+" "+y);break;
		case 06:
			System.out.print("June"+" "+d+" "+y);break;
		case 07:
			System.out.print("July"+" "+d+" "+y);break;
		case 8:
			System.out.print("August"+" "+d+" "+y);break;
		case 9:
			System.out.print("September"+" "+d+" "+y);
			break;
		case 10:
			System.out.print("October"+" "+d+" "+y);
		case 11:
			System.out.print("November"+" "+d+" "+y);break;
		case 12:
			System.out.print("December"+" "+d+" "+y);break;
			default: System.out.println("Not a valid month number");break;
		}
		

	}

}

File: answers/Gravity4543/conditionals/date.c
#include <stdio.h>

int main()
{
    int date, month, year;
    printf("enter the date in dd/mm/yyyy format: ");
    scanf("%d/%d/%d", &date, &month, &year);
    printf("%d", date);
    // date with if else
    if (date == 1 || date == 21 || date == 31)
    {
        printf("st ");
    }
    else if (date == 2 || date == 22)
    {
        printf("nd ");
    }
    else if (date == 3 || date == 23)
    {
        printf("rd ");
    }
    else
    {
        printf("th ");
    }
    //writing month in words with switch case
    switch (month)
    {
    case 1:
        printf("january ");
        break;
    case 2:
        printf("february ");
        break;
    case 3:
        printf("march ");
        break;
    case 4:
        printf("april ");
        break;
    case 5:
        printf("may ");
         break;
    case 6:
        printf("june ");
        break;
    case 7:
        printf("july ");
        break;
    case 8:
        printf("august ");
        break;
    case 9:
        printf("september ");
        break;
  case 10:
        printf("october ");
        break;
     case 11:
        printf("november ");
        break;
    case 12:
        printf("december ");
        break;
    default:
        printf("error");
    }
printf("%d\n", year);
    return 0;
}